### PR TITLE
Birth form correction request e2e tests

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Add first e2e test suites for birth form correction flow

## Checklist

- [x] I have linked the correct Github issue under "Development"
- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/testcases/v2-correction-birth/birth-correction-flow.spec.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+      .getByRole('button', { name: formatV2ChildName(declaration) })
+      .click()
+    await ensureAssigned(page)
+    await selectAction(page, 'Correct record')
+  })
+
+  test('Try to continue without filling in required fields', async () => {
+    await page.getByRole('button', { name: 'Continue' }).click()
+    await expect(page.locator('#requester____type_error')).toBeVisible()
+    await expect(page.locator('#reason____option_error')).toBeVisible()
```
Comment (jamil314) [inline e2e/testcases/v2-correction-birth/birth-correction-flow.spec.ts]: Could you also assert the error message?

**Review by jamil314 — APPROVED** [review]
